packages/react/src/components/Authenticator/SignUp/FormFields.tsx

```diff
@@ -70,15 +76,29 @@ export function FormFields() {
             );
           case 'password':
             return (
-              <PasswordField
-                key={name}
-                autoComplete="new-password"
-                data-amplify-password
-                {...propsCreator('password', 'Password', formOverrides, true)}
-                hasError={!!validationError['confirm_password']}
-                name="password"
-                onBlur={handleBlur}
-              />
+              <React.Fragment key={name}>
+                <PasswordField
+                  autoComplete="new-password"
+                  data-amplify-password
+                  {...propsCreator('password', 'Password', formOverrides, true)}
+                  hasError={!!validationError['confirm_password']}
+                  name="password"
+                  onBlur={handleBlur}
+                />
+                {validationError['password'] && (
+                  <View data-amplify-sign-up-errors>
+                    {(validationError['password'] as string[])?.map(
```

Comment:
also here, not sure why you need to do those `as string` in this PR, I don't see my VS Code complains about them

Reply:
See previous comment.